host/3.0/buster/amd64/python/python36/python36-slim.Dockerfile

```diff
@@ -58,7 +72,7 @@ RUN apt-get update && \
     curl https://packages.microsoft.com/keys/microsoft.asc | apt-key add - && \
     curl https://packages.microsoft.com/config/debian/9/prod.list > /etc/apt/sources.list.d/mssql-release.list && \
     # Needed for libss1.0.0 and in turn MS SQL
-    echo 'deb http://security.debian.org/debian-security jessie/updates main' >> /etc/apt/sources.list && \
+    # echo 'deb http://security.debian.org/debian-security jessie/updates main' >> /etc/apt/sources.list && \
```

Comment:
What is the impact of removing it - I would ask the MS-SQL team (at least our PM working with them) to inform the use of this package.

Reply:
This line is commented in dev branch. No change from my side for this PR.